include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -10065,6 +10065,10 @@
         {
           "enumerant" : "SYCL",
           "value" : 7
+        },
+        {
+          "enumerant" : "C",
```

Comment:
This addition was discussed during the last SPIR-V call today, here's a summary:

We have already various language entries for OpenCL C and C++ flavours, so perhaps "C" is a bit too generic. Both in the sense that it appears you are implementing a subset of C, but also you have your own extensions for exposing things like textures etc which do not exist in standard C. Perhaps it makes sense to define your own language for your dialect of C, and create a language entry for that ?